sport/app/football/controllers/model.scala

```diff
@@ -40,29 +39,7 @@ class FootballPage(
    *
    * Has to be 'def' to pick up current switch state.
    */
```

Comment:
This comment can go too

Reply:
Was thinking removing it as part of the coming cleaning.